src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="img">
+        <a
+        href="#">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav
+      class="nav">
+
+
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="menu is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="menu">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="menu">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" data-qa="hover"
+             class="menu">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="menu">
+            Gadgets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="menu">
```

Comment:
Let's fix classes everywhere
```suggestion
            <a href="#" class="nav__link">
```